<launch>
  <arg name="input" default="/vase_detection/passthrough/output"/>
  <node name="projector" pkg="tbtop_square" type="projector">
    <remap from="output" to="cloud2d_projected"/>
    <remap from="input" to="$(arg input)"/>
  </node>
</launch>
